core/controllers/base.py

```diff
@@ -190,6 +186,10 @@ def __init__(self, request, response):  # pylint: disable=super-init-not-called
             feconf.ROLE_ID_GUEST
             if self.user_id is None else user_settings.role)
         self.user = user_services.UserActionsInfo(self.user_id, self.role)
+        self.system_user = user_services.UserActionsInfo(
```

Comment:
Probably better to define this using user_services.get_system_user(). (I think this is one exception to the general rule -- it's really a constant of the system, and it just doesn't really make sense to define this for every request when it may not even get used.) Also we should avoid passing in superfluous args to load_demo() etc. if they're not needed.